### PR TITLE
[WIP] uim 1.8.6

### PR DESCRIPTION
**Pull request by L2G**
This is not ready for prime time; I'm only making a pull request to give it visibility while it's being worked on.

The version in core right now is 1.6.0 and also does not build, at least not on my 10.9 Mavericks system.

Note that there is another, older PR (#38267) that also modifies this formula (it adds a HEAD build).

Of course this will be condensed into a single commit when it's "done."

### CODE REVIEW

File: Library/Formula/uim.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  homepage 'http://code.google.com/p/uim/'
-  url 'https://uim.googlecode.com/files/uim-1.6.0.tar.bz2'
-  sha1 'd27f2ca8136da0702c82f0522911d06b2b8f8ea7'
+  homepage "http://code.google.com/p/uim/"
+  url "https://github.com/uim/uim/releases/download/uim-1.8.6/uim-1.8.6.tar.gz"
+  sha256 "0eebcf5b2192d0cd11018adbd69e358bfa062d3b5d63c1dd0b73d5f63eb7afe7"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'gettext'
+  depends_on :x11
```
Comment (mikemcquaid) [inline Library/Formula/uim.rb]: If this is mandatory this should be moved to `homebrew-x11`

Comment (L2G) [inline Library/Formula/uim.rb]: I didn't become aware of it until I started working on it. I think it will indeed have to be moved over.

Comment (mikemcquaid) [inline Library/Formula/uim.rb]: Cool, mind changing this PR to do that? Thanks!

Comment (L2G) [inline Library/Formula/uim.rb]: No problem.

**Comment by DomT4** [comment]
There's no other dependency usage in the core, so let's just tweak this PR to migrate this to X11 :+1:

**Comment by mikemcquaid** [comment]
> There's no other dependency usage in the core, so let's just tweak this PR to migrate this to X11

:+1:

**Comment by DomT4** [comment]
Ping? :)

**Comment by bfontaine** [comment]
Closing due to the inactivity. Feel free to open another PR in the future!

**Pull request closed without merging**